plans/verify/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"net"
+	"strings"
+	"time"
+
+	"github.com/ipfs/testground/sdk/runtime"
+	"github.com/ipfs/testground/sdk/sync"
+	"github.com/sparrc/go-ping"
+)
+
+func main() {
+	testcases := map[string]runtime.TestCaseFn{
+		"uses-data-network": UsesDataNetwork,
+	}
+	runtime.InvokeMap(testcases)
+}
+
+func setupNetwork(ctx context.Context, runenv *runtime.RunEnv) (*sync.Client, error) {
+	client := sync.MustBoundClient(ctx, runenv)
+	return client, client.WaitNetworkInitialized(ctx, runenv)
+}
+
+func isControlNet(nw string) bool {
```

Comment:
I know I can parse and check if the address is in range rather than using HasPrefix. 

Is there a better way for determining the range for k8s control networks? It might not always be this /24,

Reply:
@coryschwartz please extract those as `const`s at the top of the file.